projects/hslayers/src/components/query/query.component.ts

```diff
@@ -91,7 +94,18 @@ export class HsQueryComponent {
       );
     });
   }
-
+  checkForBodyElements(docChildren: any): any[] {
```

Comment:
Wouldnt it be possible to use just getElementsByTagName() or querySelectorAll()

Reply:
Well getInvisiblePopup() already does that. Rest of the code checks if there is content for the popup available, otherwise empty popup won't be shown.

Reply:
Could you use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some  on invisiblePopup.contentDocument.body.children array please? No need to have return the actual bodyElements list if we are just checking if any non-meta-head exists.